### PR TITLE
Amend preview chart for redirect config

### PR DESCRIPTION
**Pull request by dharmendrak**

### CODE REVIEW

File: charts/cmc-citizen-frontend/values.preview.template.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 tags:
-  ccd-idam-pr: true
```
Comment (klawani) [inline charts/cmc-citizen-frontend/values.preview.template.yaml:2]: I think we need to remove this completely... it's not needed anymore. It's already enabled in the chart we are using.
https://github.com/hmcts/chart-ccd/blob/master/ccd/values.yaml#L2

File: charts/cmc-citizen-frontend/values.preview.template.yaml [hunk 2]
```diff
@@ -151,6 +151,8 @@ ccd:
       keyVaults:
       environment:
         IDAM_OAUTH2_AW_CLIENT_SECRET: ${ADMIN_WEB_IDAM_SECRET}
+    idam-pr:
```
Comment (klawani) [inline charts/cmc-citizen-frontend/values.preview.template.yaml:154]: you don't really need this. it's already in ccd-chart-4 we are using

File: charts/cmc-citizen-frontend/values.preview.template.yaml (hunk 1 above)
Comment (dharmendrak) [reply charts/cmc-citizen-frontend/values.preview.template.yaml:2]: removed

File: charts/cmc-citizen-frontend/values.preview.template.yaml (hunk 2 above)
Comment (dharmendrak) [reply charts/cmc-citizen-frontend/values.preview.template.yaml:151]: I think it is needed

**Pull request merged**